### PR TITLE
[MIG] 10.0 porting hr_skill

### PR DESCRIPTION
**Pull request by jlaloux**

### CODE REVIEW

File: hr_skill/__manifest__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
```
Comment (feketemihai) [inline hr_skill/__manifest__.py:4]: @jlaloux Thanks for your contribution, can you please update with small headers, add yourself in the Contributors list...

**Comment by feketemihai** [comment]
@jlaloux See comments from #299 about the menus and views...

File: hr_skill/__manifest__.py (hunk 1 above)
Comment (jlaloux) [reply hr_skill/__manifest__.py:4]: Hi @feketemihai,
What do you mean by "update with small headers"? I don't see any python filename with big headers.
Thanks

Comment (jlaloux) [reply hr_skill/__manifest__.py:4]: I supposed this was about updating copyrights?

File: hr_skill/views/hr_skill.xml [hunk 2]
```diff
@@ -46,22 +41,8 @@
     </record>
 
     <menuitem name="Skills"
-              parent="menu_hr_skill_mgt"
+              parent="hr.menu_human_resources_configuration"
```
Comment (feketemihai) [inline hr_skill/views/hr_skill.xml:44]: @jlaloux Hi, thanks for the update, can you add a groups="base.group_no_one" to the menu, to only be visible when you activate the developer mode...

**Review by gurneyalex — APPROVED** [review]

**Review by andhit-r — CHANGES_REQUESTED** [review]
@jlaloux you may want to add unittest to pass the codecov test.

**Comment by jlaloux** [comment]
Test added

File: hr_skill/views/hr_skill.xml (hunk 2 above)
Comment (jlaloux) [reply hr_skill/views/hr_skill.xml:44]: Done ;)

File: hr_skill/views/hr_skill.xml [hunk 3]
```diff
@@ -43,6 +43,7 @@
     <menuitem name="Skills"
               parent="hr.menu_human_resources_configuration"
               id="menu_open_view_skill_form"
-              action="open_view_skill_form"/>
+              action="open_view_skill_form"
+              groups="base.group_no_one"/>
```
Comment (dreispt) [inline hr_skill/views/hr_skill.xml]: This menu item will only be visible if the parent Configuration menu item also is.
So IMO this change in unnecessary.

File: hr_skill/__manifest__.py (hunk 1 above)
Comment (dreispt) [reply hr_skill/__manifest__.py:4]: It's OK as it is: porting a module does not grant copyright on it.

File: hr_skill/views/hr_skill.xml (hunk 2 above)
Comment (dreispt) [reply hr_skill/views/hr_skill.xml:44]: @feketemihai AFAICT that won't be needed - the menu is only visible if the parent id.

**Review by dreispt — COMMENTED** [review]
LGTM, only a comment about the menu items.

**Review by leemannd — APPROVED** [review]
Thank you for the port.
LGTM
Travis error doesn't seems to be related. 
@jlaloux Could you rebase please?

File: hr_skill/tests/test_hr_skill.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+from odoo.tests.common import TransactionCase
+
+
+class TestHrSkill(TransactionCase):
+
+    def test_name_get(self):
+        hr_skill_model = self.env['hr.skill']
+        soft_skill_id = hr_skill_model.create({'name': 'Soft skill'})
+        leadership_skill_id = hr_skill_model.create({
```
Comment (jcdrubay) [inline hr_skill/tests/test_hr_skill.py:13]: You could remove the suffix _id for `soft_skill_id` and `leadership_skill_id` as those 2 variables represent an instance of object.

**Review by jcdrubay — APPROVED** [review]
with a suggestion of variable renaming in the test.

**Comment by dreispt** [comment]
Can you please rebase on current 10.0 branch, to check if we get a green build?

**Comment by dreispt** [comment]
I'm sorry @jlaloux , but the change set includes files not in the module you are migrating (diffs in 351 files!). If you're having a hard time with `git rebase` I suggest you to start a new clean PR.

**Review by gurneyalex — CHANGES_REQUESTED** [review]
there was a problem in the rebase, as pointed by @dreispt

**Comment by leemannd** [comment]
@jlaloux If you need help for the rebase just ping me. You can go back to the original state of this PR with git reflog https://git-scm.com/docs/git-reflog

**Comment by leemannd** [comment]
I extracted the commits left in his PR to superseed https://github.com/OCA/hr/pull/378.

**Comment by pedrobaeza** [comment]
Superseeded by #378

**Pull request closed without merging**